tests/post_training/test_templates/test_quantizer_config.py

```diff
@@ -85,10 +85,27 @@ class TestGetStatisticsCollectorParameters:
         ref_per_ch_reduction_axes: List[int]
         ref_per_tensor_reduction_axes: List[int]
 
-    @abstractmethod
-    @pytest.fixture
+    @pytest.fixture(
+        params=[
+            pytest.param(
+                TestGetStatisticsCollectorParameters(TargetType.PRE_LAYER_OPERATION, "/Sum_1_0", (2,), (1, 2)),
+            ),
+            TestGetStatisticsCollectorParameters(
+                TargetType.POST_LAYER_OPERATION,
+                "/Conv_1_0",
+                (2, 3),
+                (1, 2, 3),
+            ),
+            TestGetStatisticsCollectorParameters(
+                TargetType.OPERATION_WITH_WEIGHTS,
+                "/Conv_1_0",
+                (1, 2, 3),
+                (0, 1, 2, 3),
```

Comment:
Where are the `None` cases?

Reply:
This test was not introduced in that PR

Reply:
That is true. But you've replaced it incorrectly. Please, see the `tests/openvino/native/quantization/test_quantizer_config.py` for example.

Reply:
The values were updated. Now all backend values aligned